src/chart/sunburst/SunburstPiece.ts

```diff
@@ -219,7 +219,13 @@ class SunburstPiece extends graphic.Sector {
             }
             else {
                 if (!textAlign || textAlign === 'center') {
-                    r = (layout.r + layout.r0) / 2;
+                    // set the position of label in center when angle is 2π & r0 is 0
```

Comment:
It will be more clear to be commented 
```
// Put label in the center if it's a circle.
```
Current comment is just an plain explaination on the code.